src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import classNames from 'classnames';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  filterValue: FilterType;
+  setFilter: (value: FilterType) => void;
+  onClear: () => void;
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  filterValue,
+  setFilter,
+  onClear,
+}) => {
+  const completedTodos = todos.filter(todo => todo.completed);
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const filters = Object.values(FilterType);
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      {/* Hide the footer if there are no todos */}
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        {filters.map((value, i) => (
+          <a
+            key={i}
```

Comment:
Do you need index here?
```suggestion
        {filters.map((value) => (
          <a
            key={value}
```